libavfilter/dnn/dnn_backend_native_layer_conv2d.c

```diff
@@ -33,13 +34,25 @@ typedef struct thread_common_param{
     const void *parameters;
     NativeContext *ctx;
     int thread_num;
+    int step;
 } thread_common_param;
 
 typedef struct thread_param{
     thread_common_param *thread_common_param;
     int thread_index;
 } thread_param;
 
+typedef struct execute_data{
+    int thread_start, thread_end, input_num, output_num, kernel_size, padding_method, dilation;
+    int pad_size, width, height, radius, src_linesize, filter_size, filter_linesize;
+    float *input;
+    float *output;
+    float *kernel;
+} execute_data;
+
+void ff_dnn_execute_layer_conv2d_sse4(execute_data *execute_data);
+void ff_dnn_execute_layer_conv2d_c(execute_data *execute_data);
```

Comment:
please use two patches, one patch for separate function, the other patch to add sse4 support.